github3/events.py

```diff
@@ -30,6 +33,9 @@ class Event(GitHubCore):
     """
 
     def _update_attributes(self, event):
+        # The _json_data is modified here
+        self._orig_json_data = copy.deepcopy(event)
```

Comment:
Since event is the same thing as `_json_data` what I was looking for was just a simple:

```py
# If we don't copy this, then we end up altering _json_data which we do not want to do:
event = copy.deepcopy(event)
```

That should more than suffice for what we need.